### PR TITLE
Add privacy-services-manager (new cask)

### PR DESCRIPTION
**Pull request by jawshooah**
Refs #9068

### CODE REVIEW

File: Casks/privacy-services-manager.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'privacy-services-manager' do
+  version '1.5.0'
+  sha256 'fde9160728499ee1a24171b09f160b1e55892b59d8af7293ee3c5205766eb886'
+
+  url "https://github.com/univ-of-utah-marriott-library-apple/privacy_services_manager/releases/download/#{version}/Privacy_Services_Management_#{version}.dmg"
+  name 'Privacy Services Manager'
+  name 'Privace Services Management'
```
Comment (vitorgalvao) [inline Casks/privacy-services-manager.rb]: Is this one intended?

Comment (jawshooah) [inline Casks/privacy-services-manager.rb]: All branding points to the name "Privacy Services Manager", but the pkg installer is named "Privacy Services Management". Figured it couldn't hurt to have both.

Comment (vitorgalvao) [inline Casks/privacy-services-manager.rb]: I was referring to the typo (was trying to make sure if it had been made by you or them). You wrote “Privac**e**”.

Comment (jawshooah) [inline Casks/privacy-services-manager.rb]: Ahh, didn't catch that. Haven't slept much.

**Pull request merged**